addon/authenticators/jwt.js

```diff
@@ -103,7 +104,7 @@ export default TokenAuthenticator.extend({
       dataObject = Ember.Object.create(data);
 
     return new Ember.RSVP.Promise(function(resolve, reject) {
-      var now = (new Date()).getTime();
+      var now = _this.getCurrentTime();
```

Comment:
@RuslanZavacky Why is `getCurrentTime()` here useful? Why would you override it?